src/Ocelot/Configuration/SecurityOptions.cs

```diff
@@ -1,17 +1,43 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
+using NetTools;
 
 namespace Ocelot.Configuration
 {
     public class SecurityOptions
     {
-        public SecurityOptions(List<string> allowedList, List<string> blockedList)
+        public SecurityOptions(List<string> allowedList, List<string> blockedList, bool excludeAllowedFromBlocked)
```

Comment:
In theory it can be a breaking change!
So, I would prefer to define 2 constructors:
- old constructor with 2 parameters
- new constructor with 3 parameters

Finally, reuse old constructor in the new one via `this(1, 2)`

Reply:
Yes, it is!